infra/terraform/modules/data_access/organization_events/organization_events.py

```diff
@@ -0,0 +1,53 @@
+import json
+import logging
+import os
+
+import boto3
+
+
+'''
+Environment variables:
+ - CREATE_ROLE_ARN, ARN of the lambda function to create the IAM role
+ - DELETE_ROLE_ARN, ARN of the lambda function to delete the IAM role
+ - LOG_LEVEL, change the logging level (default is "DEBUG"). Must be one of
+   the python logging supported levels: "CRITICAL", "ERROR", "WARNING",
+   "INFO" or "DEBUG" (See: https://docs.python.org/2/library/logging.html#logging-levels)
+'''
+
+
+LOG = logging.getLogger(__package__)
+LOG_LEVEL = os.environ.get("LOG_LEVEL", "DEBUG")
+LOG.setLevel(LOG_LEVEL)
+
+
+def event_received(sns_event, context):
+    EVENT_HANDLERS = {
+        "member_added": os.environ["CREATE_ROLE_ARN"],
+        "member_removed": os.environ["DELETE_ROLE_ARN"],
+    }
+
+    LOG.debug("SNS event received = {}".format(json.dumps(sns_event)))
+    for record in sns_event["Records"]:
+        event = json.loads(record["Sns"]["Message"])
+        action = event["action"]
+
+        if action in EVENT_HANDLERS:
+            invoke_lambda(
+                function=EVENT_HANDLERS[action],
+                payload=payload(event)
+            )
+
+
+def invoke_lambda(function, payload):
+    client = boto3.client("lambda")
+    client.invoke(
+        FunctionName=function,
+        Payload=bytes(json.dumps(payload), "utf8"),
+        InvocationType="Event",
+    )
+
+
+def payload(event):
+    return {
+        "username": event["membership"]["user"]["login"],
+    }
```

Comment:
I like to wrap things in small function especially when used 2+ times but after the refactored we use this only once so I guess I can just inline it as it's relatively simple 👍